metricbeat/module/ceph/_meta/config.yml

```diff
@@ -0,0 +1,25 @@
+- module: ceph
+  metricsets: ["perf","status","df","osdpoolstats"]
+  enabled: true
+  period: 5s
+
+  ceph:  
+  # location of ceph binary
```

Comment:
I suggest to split up the config into config.full.yml and config.yml and only have the extensive options in the full file. See other metricsets for examples.